templates/types/streaming/fastapi/main.py

```diff
@@ -38,10 +38,8 @@ async def redirect_to_docs():
         return RedirectResponse(url="/docs")
 
 
-if os.path.exists("data"):
-    app.mount("/api/data", StaticFiles(directory="data"), name="static")
-
-app.mount("/api/tool-output", StaticFiles(directory="tool-output"), name="static")
+app.mount("/api/files/data", StaticFiles(directory="data"), name="data-static")
+app.mount("/api/files/tool-output", StaticFiles(directory="tool-output"), name="tool-output-static")
```

Comment:
don't remove the path check and don't add tool output yet (no python support in this PR)
```suggestion
if os.path.exists("data"):
    app.mount("/api/files/data", StaticFiles(directory="data"), name="data-static")
```